src/ui/inputwin.c

```diff
@@ -924,6 +927,45 @@ _inp_rl_subwin_pagedown_handler(int count, int key)
     return 0;
 }
 
+static int
+_inp_rl_scroll(int count, int key)
+{
+    if (!prefs_get_boolean(PREF_MOUSE))
+        return 0;
+    cons_show(g_strdup_printf("scroll. count: %d, key: %d, %c", count, key, key));
```

Comment:
I guess this in only a debug statement, which will go soonish.

FYI `cons_show()` behaves like `printf()`, i.e. you can drop the call to `g_strdup_printf()`. (also you're leaking the value returned :D)